src/transformers/tokenization_utils_base.py

```diff
@@ -2417,6 +2424,22 @@ def _from_pretrained(
                 "Special tokens have been added in the vocabulary, make sure the associated word embeddings are"
                 " fine-tuned or trained."
             )
+        if tokenizer.vocab_size > 100000 and getattr(tokenizer.backend_tokenizer, "pre_tokenizer", None) is not None:
+            from huggingface_hub import model_info
+            def is_base_mistral(model_id: str) -> bool:
+                model = model_info(model_id)
+                if model.tags is not None:
+                    if re.search("base_model:.*mistralai", "".join(model.tags)):
+                        return True
+                return False
+
+            if is_base_mistral(pretrained_model_name_or_path) and not kwargs.get("fix_regex"):
+                logger.warning(
+                    f"The tokenizer you are loading from '{pretrained_model_name_or_path}'"
+                    f" with an old regex pattern. This will lead to incorrect tokenization."
```

Comment:
```suggestion
                    f" with an incorrect regex pattern. This will lead to incorrect tokenization on some edge cases. See issue: ....."
```
let's maybe say incorrect no?